src/transformers/models/clip/modeling_clip.py

```diff
@@ -977,7 +977,7 @@ def __init__(self, config: CLIPConfig):
 
         self.visual_projection = nn.Linear(self.vision_embed_dim, self.projection_dim, bias=False)
         self.text_projection = nn.Linear(self.text_embed_dim, self.projection_dim, bias=False)
-        self.logit_scale = nn.Parameter(torch.ones([]) * self.config.logit_scale_init_value)
+        self.logit_scale = nn.Parameter(torch.tensor(1.0) * self.config.logit_scale_init_value)
```

Comment:
Let's change all such occurrences? Most of them are in clip-family models.

Reply:
Totally!

Reply:
done